app/src/NativeBase/Containers/VerticalStackContainer.tsx

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
Is this an example container to show how to do one?  If so, wonder if we should remove it -- or if it's helpful to leave it in for now, do we need another ticket to make sure we take it out later?

Reply:
It's a good demo example, I think worth leaving in for demo purposes for now until we've got more of the codebase moved over. Good shout to raise a ticket to remove it later though; I'll raise that now.

Reply:
Nice one, thanks 👍